ui/apps/platform/src/Containers/MainPage/InviteUsersForm.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactElement } from 'react';
+import { Form, SelectOption, TextArea } from '@patternfly/react-core';
+
+import FormLabelGroup from 'Components/PatternFly/FormLabelGroup';
+import SelectSingle from 'Components/SelectSingle';
+
+function InviteUsersForm({ formik, providers, roles, onChange }): ReactElement | null {
+    const { values, touched, errors, handleBlur } = formik;
+
+    return (
+        <Form>
+            <FormLabelGroup
+                isRequired
+                label="Emails to invite"
+                fieldId="emails"
+                touched={touched}
+                errors={errors}
+                helperText="Multiple emails should be separated with commas."
+            >
+                <TextArea
+                    isRequired
+                    className="json-input"
```

Comment:
Will this class get picked up by the `IntegrationForm.css` file? From what I can tell that CSS may or may not be loaded for this form so the intended style might not always be present.

Reply:
Ah, this was a copy-paste miss. I grabbed a text area from the an integration form.

If you go to the integration section first, then the CSS will persist, but I will remove that class, because it's not needed.